src/actions/MetadataActions.js

```diff
@@ -6,14 +6,19 @@ class MetadataActions {
     this.generateActions(
       'createSecretSuccess',
       'deleteSecretSuccess',
+      'deleteSecretFailure',
       'createSecretUserRightsSuccess',
       'createSecretUserRightsFailure',
       'updateSecretUserRightsSuccess',
       'updateSecretUserRightsFailure',
       'deleteSecretUserRightsSuccess',
       'deleteSecretUserRightsFailure',
+      'removeSecretFromCurrentFolderSuccess',
+      'removeSecretFromCurrentFolderFailure',
       'moveSecretToFolderSuccess',
       'moveSecretToFolderFailure',
```

Comment:
Please cleanup unused/commented code. Even if we plan to re-use it later, git is here to retrieve it if we need :)